--- mpih-add1.S.orig	2017-09-29 13:49:31.000000000 -0700
+++ mpih-add1.S	2017-09-29 13:50:52.000000000 -0700
@@ -34,7 +34,7 @@
 .text
 
 .globl _gcry_mpih_add_n
-.type  _gcry_mpih_add_n,%function
+// .type  _gcry_mpih_add_n,%function
 _gcry_mpih_add_n:
 	and	x5, x3, #3;
 	adds	xzr, xzr, xzr; /* clear carry flag */
@@ -68,4 +68,4 @@
 .Lend:
 	adc	x0, xzr, xzr;
 	ret;
-.size _gcry_mpih_add_n,.-_gcry_mpih_add_n;
+// .size _gcry_mpih_add_n,.-_gcry_mpih_add_n;
